packages/admin/cms-admin/src/builds/PublisherPage.tsx

```diff
@@ -0,0 +1,135 @@
+import { gql } from "@apollo/client";
+import {
+    LocalErrorScopeApolloContext,
+    messages,
+    Stack,
+    Toolbar,
+    ToolbarActions,
+    ToolbarFillSpace,
+    ToolbarTitleItem,
+    useTableQuery,
+} from "@comet/admin";
+import { Domain } from "@comet/admin-icons";
+import { Typography } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import { DataGrid } from "@mui/x-data-grid";
+import { parseISO } from "date-fns";
+import * as React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import { ContentScopeIndicator } from "../contentScope/ContentScopeIndicator";
+import { GQLBuildsQuery } from "../graphql.generated";
+import { BuildRuntime } from "./BuildRuntime";
+import { PublishButton } from "./PublishButton";
+
+const ScopeIndicatorLabelBold = styled(Typography)`
+    && {
+        font-weight: 400;
+        padding: 0 8px 0 4px;
+        text-transform: uppercase;
+    }
+`;
+
+const ScopeIndicatorContent = styled("div")`
+    display: flex;
+    align-items: center;
+`;
+
+const buildsQuery = gql`
+    query Builds {
+        builds {
+            id
+            status
+            name
+            trigger
+            startTime
+            completionTime
+        }
+    }
+`;
+
+const DataGridContainer = styled("div")`
+    width: 100%;
+    height: calc(100vh - var(--comet-admin-master-layout-content-top-spacing));
+`;
+
+export function PublisherPage(): React.ReactElement {
+    const intl = useIntl();
+
+    const { tableData, loading, error } = useTableQuery<GQLBuildsQuery, undefined>()(buildsQuery, {
```

Comment:
Please do not use `useTableQuery` in combination with Data Grid. Use the new Data Grid helpers instead: https://storybook.comet-dxp.com/?path=/story/docs-components-datagrid--page